consensus/polybft/extra.go

```diff
@@ -326,10 +326,9 @@ func (s *Signature) VerifyCommittedFields(validatorSet AccountSet, hash types.Ha
 	}
 
 	if len(filtered) < getQuorumSize(validatorSet.Len()) {
-		return fmt.Errorf("quorum not reached")
+		return fmt.Errorf("quorum not reached: %d of %d", len(filtered), getQuorumSize(validatorSet.Len()))
```

Comment:
Nitpick: You can extract this `getQuorumSize(validatorSet.Len())` to a local variable, and use it in both condition and log.

Reply:
fixed https://github.com/0xPolygon/polygon-edge/commit/f0d08eb1a44c7b7b182c3f1757dc2e855bee7d57